strr-examiner-web/app/stores/examiner.ts

```diff
@@ -31,6 +31,26 @@ export const useExaminerStore = defineStore('strr/examiner-store', () => {
   const sendNocSchema = computed(() => z.object({
     content: z.string().min(1, { message: t('validation.nocContent') })
   }))
+  const defaultApplicationStatuses = [
```

Comment:
What are these default statuses for?

Reply:
If no status is explicitly selected then the applications of these statuses will be shown by default.
https://github.com/bcgov/STRR/pull/658

Reply:
Should we just define which statuses not to show by default?

Reply:
The API filter works based on inclusion, so explicit statuses need to be included when calling the endpoint.